java/client/src/main/java/glide/api/commands/servermodules/GlideJson.java

```diff
@@ -0,0 +1,411 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands.servermodules;
+
+import static glide.api.models.GlideString.gs;
+
+import glide.api.BaseClient;
+import glide.api.GlideClient;
+import glide.api.GlideClusterClient;
+import glide.api.models.ClusterValue;
+import glide.api.models.GlideString;
+import glide.api.models.commands.ConditionalChange;
+import glide.api.models.commands.json.JsonGetOptions;
+import glide.utils.ArgsBuilder;
+import glide.utils.ArrayTransformUtils;
+import java.util.concurrent.CompletableFuture;
+import lombok.NonNull;
+
+public class GlideJson {
+
+    public static final String JSON_PREFIX = "JSON.";
+    public static final String JSON_SET = JSON_PREFIX + "SET";
+    public static final String JSON_GET = JSON_PREFIX + "GET";
+
+    private GlideJson() {}
```

Comment:
huh?

Reply:
This is the Java way to make the class static, so no instance of it can ever be created.